code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -26,14 +26,32 @@
 	var/datum/action/innate/slime_pick_up/slime_up_action = new
 	var/datum/action/innate/feed_slime/feed_slime_action = new
 	var/datum/action/innate/monkey_recycle/monkey_recycle_action = new
+	var/datum/action/innate/slime_scan/scan_action = new
+	var/datum/action/innate/feed_potion/potion_action = new
+	var/datum/action/innate/hotkey_help/hotkey_help = new
 
 	var/list/stored_slimes = list()
 	var/max_slimes = 5
 	var/monkeys = 0
+	var/obj/item/slimepotion/slime/current_potion
+	var/obj/machinery/monkey_recycler/connected_recycler
 
 	icon_screen = "slime_comp"
 	icon_keyboard = "rd_key"
 
+/obj/machinery/computer/camera_advanced/xenobio/New()
```

Comment:
You're still not calling parent.

Reply:
could you please elaborate

Reply:
you need to call parent so that all parent object's `New` occur, as well...like being added to the machine list, using power, etc.

`..()`